vyper/functions/functions.py

```diff
@@ -625,6 +625,13 @@ def _max(expr, args, kwargs, context):
 
 
 def minmax(expr, args, kwargs, context, is_min):
+    def _can_compare_with_uint256(operand):
+        if operand.typ.typ == 'uint256':
+            return True
+        elif operand.typ.typ == 'int128' and operand.typ.is_literal and operand.value >= 0:
```

Comment:
Replace operand.value > 0 with the utils function (there is obviously an upper bound to account for as well :smiling_imp: ):
```python
operand.typ.typ == 'int128' and operand.typ.is_literal and SizeLimits.in_band('uint256', operand.value):
```